autogpt_platform/backend/backend/integrations/webhooks/manager_registry_test.py

```
@@ -0,0 +1,124 @@
+    Can't instantiate abstract class <X>WebhooksManager without an
+    implementation for abstract method 'validate_payload'
+
+These tests close that gap by exercising, for every registered provider, the
+same call the runtime makes.
+"""
+
+from typing import Type
+
+import pytest
+
+import backend.integrations.webhooks as webhooks
```

Ruff (0.16.1)

[warning] Use `from backend.integrations import webhooks` in lieu of alias

Replace with `from backend.integrations import webhooks`

(PLR0402)